src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/DefaultLSPDocumentMappingProvider.cs

```diff
@@ -143,13 +143,10 @@ public async override Task<Location[]> RemapLocationsAsync(Location[] locations,
                     continue;
                 }
 
-                var remappedLocation = new Location()
-                {
-                    Uri = razorDocumentUri,
-                    Range = remappedRange,
-                };
+                location.Uri = razorDocumentUri;
+                location.Range = remappedRange;
 
-                remappedLocations.Add(remappedLocation);
+                remappedLocations.Add(location);
```

Comment:
Previous to this change, we were throwing away information. Downstream servers could send back `VSInternalLocation` or `VSLocation`, both of which inherit from `Location` and add extra info, but if we remapped the range we'd throw the other bits of information away.

I'm not aware of a repro for this specifically, I just saw it and figured it was a pretty bad idea.